celavi/component.py

```diff
@@ -203,6 +210,11 @@ def eol_process(self, env):
                         env.now
                     )
 
+                    # TODO: consider adding a comment to explain why it is
```

Comment:
Added a comment to the TransportationTracker class regarding why that class is necessary alongside FacilityInventories.